client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/MlDataFrameAnalysisNamedXContentProvider.java

```diff
@@ -21,19 +21,17 @@
 import org.elasticsearch.common.xcontent.NamedXContentRegistry;
 import org.elasticsearch.plugins.spi.NamedXContentProvider;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class MlDataFrameAnalysisNamedXContentProvider implements NamedXContentProvider {
```

Comment:
Do we need this class at all in the client side?

Reply:
I believe we do need it at least for AbstractXContentTestCase auto-generated tests. Without this class, the parser won't know how to parse the JSON when it encounters "outlier_detection".